### PR TITLE
#163388238 upgrade django from version 1.10 to 2.1

### PR DESCRIPTION
**Pull request by unah254**
#### What does this PR do?
Have an upgrade for the application from Django 1.10 to Django 2.1
#### Description of Task to be completed?
1. Upgrade Django `pip install --upgrade django`
2. Change url to path on the url files
3. Include app_name on the url file in the authentication folder
4. Comment out Default Authentication classes on settings.py
#### How should this be manually tested?
Clone the repo and cd into it 

. checkout to the branch `git checkout ch-upgrade-django-version-163388238`

. `pip freeze` to view the dependancies

.  Django should read version 2.1

. `python manage.py runserver` to ensure server is running

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#163388238](https://www.pivotaltracker.com/n/projects/2240063)
#### Screenshots (if appropriate)
None

### CODE REVIEW

**Comment by bryan-munene** [comment]
Please gitignore and remove .vscode

File: authors/settings.py [hunk 1]
```diff
@@ -137,7 +137,7 @@
     'EXCEPTION_HANDLER': 'authors.apps.core.exceptions.core_exception_handler',
     'NON_FIELD_ERRORS_KEY': 'error',
 
-    'DEFAULT_AUTHENTICATION_CLASSES': (
-        'authors.apps.authentication.backends.JWTAuthentication',
-    ),
+    # 'DEFAULT_AUTHENTICATION_CLASSES': (
```
Comment (sam-karis) [inline authors/settings.py]: Hi good work, I don't think it necessary to comment this section of code

**Review by sam-karis — CHANGES_REQUESTED** [review]
Also fix the jwt error raised when you run the server. Can check this link: https://getblimp.github.io/django-rest-framework-jwt/

File: authors/settings.py (hunk 1 above)
Comment (unah254) [reply authors/settings.py]: Thanks Sam, the comment was necessary because we haven't configured jwt yet on `backends.py`

**Review by sam-karis — APPROVED** [review]
Good work.

**Pull request merged**